test/e2e/defaultbackend/default_backend.go

```diff
@@ -98,4 +98,35 @@ var _ = framework.IngressNginxDescribe("Default backend", func() {
 			Expect(resp.StatusCode).Should(Equal(test.Status))
 		}
 	})
+	It("should logging access to default backend when it enabled", func() {
+		f.UpdateNginxConfigMapData("enable-access-log-for-default-backend", "true")
+		host := "foo"
+		resp, _, errs := gorequest.New().
+			Get(f.IngressController.HTTPURL+"/somethingOne").
+			Set("Host", host).
+			End()
+
+		Expect(len(errs)).Should(Equal(0))
+		Expect(resp.StatusCode).Should(Equal(http.StatusNotFound))
+
+		logs, err := f.NginxLogs()
+		Expect(err).ToNot(HaveOccurred())
+		Expect(logs).To(ContainSubstring(`"/somethingOne"`))
+	})
+
+	It("should not logging access to default backend when it disabled", func() {
```

Comment:
Please change the title to something like

`disables access logging for default backend`

Reply:
Thanks, I fixed it too